docker-compose.yml

```diff
@@ -484,6 +484,7 @@ services:
         target: /home/grapl/pulumi-outputs
         read_only: false
     environment:
+      TAG:
```

Comment:
Should this have a value? What does it mean to just specify the key in this mapping?

Reply:
It just passes whatever the value of that env var, in the docker-compose execution env, to the container env. We do the same this in other places but with the alternate yaml format: https://github.com/grapl-security/grapl/blob/main/docker-compose.yml#L383. I didn't wanna change from the format used already by this service. I kind of like it better anyway.

https://docs.docker.com/compose/compose-file/compose-file-v3/#environment

Reply:
👍